app/src/main/java/io/appium/uiautomator2/handler/GetElementScreenshot.java

```diff
@@ -42,27 +43,37 @@ public AppiumResponse safeHandle(IHttpRequest request) {
                 Logger.error("Element is not visible");
                 return new AppiumResponse(getSessionId(request), WDStatus.ELEMENT_NOT_VISIBLE);
             }
-            final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
-            final File outputFile = File.createTempFile("screenshot", ".png",
-                    context.getCacheDir());
+            final Bitmap screenshot = uia.takeScreenshot();
```

Comment:
Can we extract this dancing with screenshots into separate methods to do `handlers` more readable and avoid code duplication?

Reply:
there is, actually, nothing much to extract. expect of this single line.